src/Microsoft.ML.Transforms/ExtensionsCatalog.cs

```diff
@@ -32,6 +32,31 @@ public static class MissingValueIndicatorCatalog
             => new MissingValueIndicatorEstimator(CatalogUtils.GetEnvironment(catalog), inputColumn, outputColumn);
     }
 
+    public static class MissingValueReplacerCatalog
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="MissingValueReplacingEstimator"/>
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="inputColumn">The name of the input column.</param>
+        /// <param name="outputColumn">The optional name of the output column,
+        /// If not provided, the <paramref name="inputColumn"/> will be replaced with the results of the transforms.</param>
```

Comment:
>inputColumn [](start = 49, length = 11)

Is this correct? The input column will be replaced with the results? Or should this be outputColumn?

Reply:
it is correct. The inputColumn will be overriten:


  public MissingValueReplacingEstimator(IHostEnvironment env, string inputColumn, string outputColumn = null, NAReplaceTransform.ColumnInfo.ReplacementMode replacementKind = Defaults.ReplacementMode)
            : this(env, new NAReplaceTransform.ColumnInfo(outputColumn ?? inputColumn, inputColumn, replacementKind))

---
In reply to: [230155013](https://github.com/dotnet/machinelearning/pull/1494#discussion_r230155013) [](ancestors = 230155013)